thoth/cfg.py

```diff
@@ -118,8 +133,6 @@ def _generate_basicblocks(self, instructions):
                 new_bb = True
 
             else:
-                # Should never happen
-                # all function finish by RET
```

Comment:
why removing those comments?

Reply:
i moved them to the documentation 
`        """Generate the internal list of BasicBlock
        Args:
            instructions (List): List of instructions

        Raises:
            AssertionError: Should never happen, all function finish by RET
        """`